salt/modules/status.py

```diff
@@ -1090,16 +1093,16 @@ def linux_netstats():
                     headers = comps
         return ret
 
-    def freebsd_netstats():
+    def bsd_netstats():
```

Comment:
We can't rename this function without providing some kind of alias here, since this has been released and people will be using the function. We don't want to incur an API change without handling it appropriately.

Reply:
Right; I've fixed that up by keeping the previous `freebsd_netstats()` definition for backwards compat but actually use the new `bsd_netstats()` on FreeBSD.